superset-frontend/package.json

```diff
@@ -61,47 +61,46 @@
     "@babel/runtime-corejs3": "^7.8.4",
     "@data-ui/sparkline": "^0.0.54",
     "@emotion/core": "^10.0.28",
-    "@superset-ui/chart": "^0.14.9",
-    "@superset-ui/chart-composition": "^0.14.9",
-    "@superset-ui/chart-controls": "^0.14.22",
-    "@superset-ui/color": "^0.14.9",
-    "@superset-ui/connection": "^0.14.19",
-    "@superset-ui/core": "^0.14.20",
-    "@superset-ui/dimension": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-calendar": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-chord": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-country-map": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-event-flow": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-force-directed": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-heatmap": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-histogram": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-horizon": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-iframe": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-map-box": "^0.14.9",
-    "@superset-ui/legacy-plugin-chart-markup": "^0.14.9",
```

Comment:
Bycatch: `legacy-plugin-chart-iframe` and `legacy-plugin-chart-markup` are removed here as per #10590